dev/scripts/loc.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Usage: `./loc.sh [<start_date> (default: 01-01-%Y)]`
+
+# The list of authors whose names must be merged or who must be renamed.
+# "<main name>:<name to be merged>"
+merge_list=(
+    "Adrian Richter:peb-adr"
+    "Bastian Rihm:Intevation Common Account"
```

Comment:
Sure that only @bastianjoel  used this name? :D

Reply:
In the last year, yes :D